SwrveSDK/src/amazon/java/com/swrve/sdk/SwrveEmpty.java

```diff
@@ -18,4 +19,12 @@ protected SwrveEmpty(Context context, String apiKey) {
     //@Override
     //public void setPushNotificationListener(ISwrvePushNotificationListener pushNotificationListener) {
     //}
+
+    @Override
+    public void iapPlay(String productId, double productPrice, String currency, String purchaseData, String dataSignature) {
```

Comment:
iapPlay is in ISwrveBase - didn't want to push it down to the ISwrve interface in the google flavour in case it affected much other code. Could do?